tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -42,13 +42,15 @@ spec:
       MEDIUM_STATEFUL_SETS_PER_NAMESPACE: 0
       # we are not testing PVS at this point
       CL2_ENABLE_PVS: false
+      PROMETHEUS_SCRAPE_APISERVER_ONLY: true
       PROMETHEUS_SCRAPE_KUBE_PROXY: false
       ENABLE_SYSTEM_POD_METRICS: false
       NODE_MODE: master 
       EOL
+      cat $(workspaces.source.path)/overrides.yaml
```

Comment:
Do we need this for debugging?
Can we include this info in the results uplaoded to S3?

Reply:
yes, we need those for debugging. Good point, we need to upload that to s3. It was a to-do on my end.

Reply:
Below line already adds that. I think your requirement comment is obsolete dude ^^^

Reply:
Sorry not clear which line uploads the `overrides.yaml` to S3? Is it line # 84 `aws s3 cp . s3://$(params.results-bucket)/ --recursive` ?

Reply:
Yes, upload step.